spec/views/content_items/index.html.erb_spec.rb

```diff
@@ -5,24 +5,31 @@
 
   before { assign(:content_items, content_items) }
 
-  it 'render the table header' do
+  it 'renders the table header with the right headings' do
     render
 
-    expect(rendered).to have_selector('table thead', text: 'Content Ids')
+    expect(rendered).to have_selector('table thead tr:first-child', text: 'Content Ids')
+    expect(rendered).to have_selector('table thead tr:nth(1)', text: 'Content URL')
   end
 
   it 'renders a row per Content Item' do
     render
 
-    expect(rendered).to have_selector('table tbody tr', count: 2)
+    expect(rendered).to have_selector('table tbody tr', count: 4)
   end
 
   describe 'row content' do
-    it 'includes the content-id' do
+    it 'contains the content-id in the first column' do
       content_items[0].content_id = 'a-content-id'
       render
 
-      expect(rendered).to have_selector('table tbody tr td', text: 'a-content-id')
+      expect(rendered).to have_selector('table tbody tr:first-child td', text: 'a-content-id')
+    end
+
+    it 'contains the content\'s url in the second column' do
+      render
+
+      expect(rendered).to have_selector('table tbody tr td:nth(1) a', text: 'https://gov.uk/content/item/path')
```

Comment:
Yeah, I see your point.. or we can generate like:

```ruby 
sequence(:link) { |index|  "/content/#{index}/path" }
```

This way it will be based on the index..  `build(:content_item)` `link` will be `/content/1/path`